assets/messages.yml

```diff
@@ -308,53 +353,53 @@ messages:
 
         That ticket has already been resolved as Fixed. The fix will arrive in the next version or is already included in the latest development version of the game, you can check the Fix Version/s field in that ticket to learn more.
 
-        If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been mentioned.
+        %search_for_dupes%
 
         %quick_links%
       fillname:
         - Enter parent issue ID
-  duplicate-of-mc-128302:
-    - project:
-        - mc
-        - mcl
-      name: Duplicate of MC-128302
-      shortcut: ogl
+  duplicate-of-mc-297:
+    - project: mc
+      name: MC-297 / Pixel format not accelerated
+      shortcut: '297'
       message: |-
         *Thank you for your report!*
-        We're actually already tracking this issue in *MC-128302*, so I've resolved and linked this ticket as a duplicate.
+        We're actually already tracking this issue in *MC-297*, so this ticket is being resolved and linked as a *duplicate*.
 
         This is known to be a problem with your computer.
         -- Update your [graphics card drivers|https://help.minecraft.net/hc/en-us/articles/360029643912--Updating-video-card-drivers]. (Do not rely on automatic updates)
         -- If you are using Java arguments to increase the amount of memory, please reduce it to the default 512MB.
-        -- If that did not help, please contact the *[Community Support|https://discord.gg/58Sxm23]*.
+        -- If that did not help, please contact *[Community Support|https://discord.gg/58Sxm23]*.
 
         %quick_links%
       fillname: []
-  duplicate-of-mc-297:
-    - project: mc
-      name: Duplicate of MC-297
-      shortcut: '297'
+  duplicate-of-mc-128302:
+    - project:
+        - mc
+        - mcl
+      name: MC-128302 / OpenGL not supported
+      shortcut: ogl
       message: |-
         *Thank you for your report!*
-        We're actually already tracking this issue in *MC-297*, so I've resolved and linked this ticket as a duplicate.
+        We're actually already tracking this issue in *MC-128302*, so this ticket is being resolved and linked as a *duplicate*.
 
         This is known to be a problem with your computer.
         -- Update your [graphics card drivers|https://help.minecraft.net/hc/en-us/articles/360029643912--Updating-video-card-drivers]. (Do not rely on automatic updates)
         -- If you are using Java arguments to increase the amount of memory, please reduce it to the default 512MB.
-        -- If that did not help, please contact the *[Community Support|https://discord.gg/58Sxm23]*.
+        -- If that did not help, please contact *[Community Support|https://discord.gg/58Sxm23]*.
 
         %quick_links%
       fillname: []
   duplicate-of-mcl-5638:
     - project: mcl
-      name: Duplicate of MCL-5638
-      shortcut: jre
+      name: MCL-5638 / Unable to locate Java runtime
```

Comment:
I did like those being sorted in the box together (because all duplicate messages used to start with "Duplicate".

But tbh, the dropdown becomes more and more crowded anyways, making it harder to find stuff, and we should probably think about a better solution in general.